doc/tutorial/basic/index.md

```diff
@@ -365,16 +350,40 @@ Before we start writing tests which we will be able to run through the sbt conso
 issue. Remember the task `run`? If you try to invoke it now, you will see something like this:
 
     > run
-    [info] Running tutorial.webapp.TutorialApp
-    [error] TypeError: (0 , $m_Lorg_scalajs_jquery_package$(...).jQuery$1) is not a function
-    [error]     at $c_Ltutorial_webapp_TutorialApp$.main__AT__V (.../TutorialApp.scala:9:11)
-    [error]     ...
-    [trace] Stack trace suppressed: run last compile:run for the full output.
-    [error] (compile:run) org.scalajs.jsenv.ExternalJSEnv$NonZeroExitException: Node.js exited with code 1
-    [error] Total time: 1 s, completed Oct 13, 2016 3:06:00 PM
+    [info] Running tutorial.webapp.TutorialApp. Hit any key to interrupt.
+    (...)/scalajs-tutorial/target/scala-2.13/scala-js-tutorial-fastopt.js:822
+        this.Lorg_scalajs_jquery_package$__f_jQuery = jQuery
+                                                      ^
+
+    ReferenceError: jQuery is not defined
+        at new $c_Lorg_scalajs_jquery_package$ (.../jquery/package.scala:6:41)
+        (...)
+    [error] org.scalajs.jsenv.ExternalJSRun$NonZeroExitException: exited with code 1
+    [error]         at org.scalajs.jsenv.ExternalJSRun$$anon$1.run(ExternalJSRun.scala:186)
+    [error] stack trace is suppressed; run last Compile / run for the full output
+    [error] (Compile / run) org.scalajs.jsenv.ExternalJSRun$NonZeroExitException: exited with code 1
+    [error] Total time: (...)
+
+The issue we encounter is that, although we included `jquery.js` in the HTML file, we did not instruct sbt to use when running in the command line.
+We can address this with the following additional sbt setting:
+
+{% highlight scala %}
+// Make jQuery available for `run` and `test` in sbt, *before* the Scala.js code
+val jQueryScript =
+  org.scalajs.jsenv.Input.Script(file("./jquery.js").toPath)
+Compile / jsEnvInput := jQueryScript +: (Compile / jsEnvInput).value
+Test / jsEnvInput := jQueryScript +: (Test / jsEnvInput).value
+{% endhighlight %}
```

Comment:
This is a bit problematic IMO. We really don't want anybody to think this is a good way to use Scala.js, do we?

How hard would it be to:
- Not rely on a bare JS library.
- Use scala-js-bundler in the tutorial.

Note that I haven't reviewed in detail yet.

Reply:
It highlights a limitation of how we set up the `jsEnvInput` setting. We should initialize it to `Nil` in the no-configuration scope, and *add* the linked file in each configuration. That way we would only have to write
```Scala
jsEnvInput += Input.Script(file("./jquery.js").toPath)
```
in this build.

We can improve that in 1.0.1, but for 1.0.0 we're stuck with this suboptimal config :(

Reply:
> Not rely on a bare JS library.

That would defeat the purpose of that entire section. Of course we can remove all the section about jQuery, and keep using the DOM until the end. Maybe that would be better?

> Use scala-js-bundler in the tutorial.

That's probably going a step too far for this tutorial, which in my opinion lays the ground knowledge of how Scala.js work, and which every Scala.js user needs to know.

But I can write an alternative story where we do that, for comparison.

Reply:
> keep using the DOM until the end. Maybe that would be better?

That was my thinking as well. It would avoid some of the boilerplate of dealing with native JS libs. We should defer to the `scalajs-bundler` documentation for that.

Reply:
I have pushed a commit that changes the tutorial to stick to standard DOM, rather than using jQuery. It goes with the following branch of the scalajs-tutorial repo: https://github.com/scala-js/scalajs-tutorial/commits/develop-1.x-no-jquery

Looking at it now, I actually think it's much better overall :)